src/Workspaces/Core/Portable/Indentation/AbstractIndentationService.AbstractIndenter.cs

```diff
@@ -17,25 +17,28 @@ namespace Microsoft.CodeAnalysis.Indentation
 {
     internal abstract partial class AbstractIndentationService<TSyntaxRoot>
     {
-        internal abstract class AbstractIndenter
+        protected struct Indenter
```

Comment:
Change file name to "AbstractIndentationService.Indenter" to match type name

Reply:
This was intentional.  Otherwise the diff looks like a full file delete/add :)  I can update this in the next PR though!